src/app/space/create/deployments/deployments-donut/deployments-donut.component.ts

```diff
@@ -11,6 +11,7 @@ import { Pods } from '../models/pods';
 import { DeploymentsService } from '../services/deployments.service';
 
 @Component({
+  encapsulation: ViewEncapsulation.None,
```

Comment:
If there is any css targeting child classes in `deployments-donut.component.less` they should be moved into `deployments-donut-chart.component.less`. I'd like the `ViewEncapsulation` to remain as `Default` for `deployments-donut.component.ts`.

As an aside, after investigation the css for `deployments-donut-chart.component.ts` needs a bit of tweaking but I can do that in a separate PR and issue.

Reply:
All other components have (or should have) view encapsulation to none, as we do not include the shared stylesheets in every component (just referenced). I would leave this as none, until it is decided whether the entire UI can/should/needs to be switched.

Reply:
@mindreeper2420 Oh. I wasn't aware of that. This isn't currently the case in the components under the deployments page. Most of them have the default view encapsulation. We can fix this though.

Reply:
If everything is working now, I wouldn't worry about it. Something that can be brought up with Joshua, as there was an issue on it (ViewEncapsulation) at one point